README.md

```diff
@@ -1 +1,105 @@
-## cryptocurrency-metrics App
\ No newline at end of file
+## cryptocurrency-metrics App
+
+## *Microverse React Project*
+
+> This educational project is to build a web application for a project that provides cryptocurrency statistics. The application will allow users to check the prices of coins and all their statistics.
+
+## Learning Objectives 🔖
+- [x] Use React documentation.
+- [x] Use React components.
+- [x] Use React props.
+- [x] Use React Router.
+- [x] Connect React and Redux.
+- [x] Handle events in a React app.
+- [x] Write unit tests with React Testing Library.
+- [x] Use styles in a React app.
+- [x] Use React life cycle methods.
+- [x] Apply React best practices and language style guides in code.
+- [x] Use store, actions and reducers in React.
+- [x] Perform a code review for a team member.
+
+## Built With 🛠️
+
+- [x] Languages: HTML, Css and JavaScript
+- [x] Linters: Stylelint and ESLint
+- [x] Front-end library: React with Redux Toolkit
+- [x] API: SpaceX API
+- [x] Code Editor: VS Code
+
+
+## Live Demo (Not available)
+
```

Comment:
- [ ] A link to the deployed project should be present in the README. It can be deployed on the application using Heroku, Netlify, or GitHub Pages.

- [ ] Also, there should be a recorded video presenting the features of the project.
  - The maximum length of the video is 5 minutes.
  - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
  - The video contains a demo of the project, highlighting all the project features.
  - The video contains a highlight about a piece of the code the student finds interesting or is proud of making.